sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -669,7 +670,7 @@ AllocaCommandBase *Scheduler::GraphBuilder::getOrCreateAllocaForReq(
     std::vector<Command *> &ToEnqueue) {
 
   AllocaCommandBase *AllocaCmd =
-      findAllocaForReq(Record, Req, Queue->getContextImplPtr());
+      findAllocaForReq(Record, Req, Queue->getContextImplPtr(), false);
```

Comment:
```suggestion
      findAllocaForReq(Record, Req, Queue->getContextImplPtr(), /*AllowConst=*/false);
```
Or something similar would be nice to have here and other cases doing this, just to be nice to the reader. 😄

Reply:
Done.